fluentui_core/src/main/java/com/microsoft/fluentui/theme/FluentTheme.kt

```diff
@@ -82,39 +86,51 @@ object FluentTheme : ViewModel() {
         @ReadOnlyComposable
         get() = LocalThemeMode.current
 
+
+    val themeID: Int
+        @Composable
+        @ReadOnlyComposable
+        get() = LocalThemeID.current
+
     private var aliasTokens_: MutableLiveData<IAliasTokens> = MutableLiveData(AliasTokens())
     private var controlTokens_: MutableLiveData<IControlTokens> = MutableLiveData(ControlTokens())
     private var themeMode_: MutableLiveData<ThemeMode> = MutableLiveData(ThemeMode.Auto)
+    private var themeID_: MutableLiveData<Int> = MutableLiveData(1)
 
     /**
      * Update aliasTokens across all FluentTheme scope where explicit values is not provided to it.
      */
     fun updateAliasTokens(overrideAliasTokens: IAliasTokens) {
         aliasTokens_.value = overrideAliasTokens
+        updateThemeID()
     }
 
     /**
      * Update controlTokens across all FluentTheme scope where explicit values is not provided to it.
      */
     fun updateControlTokens(overrideControlTokens: IControlTokens) {
         controlTokens_.value = overrideControlTokens
+        updateThemeID()
```

Comment:
More of a curiosity query, the solution propose relies on creating unique Id's per update to force a render. In other UI frameworks, I have seem alternative solutions like triggering update by creating new object instead of value updates on old instance ( so that the equality checks to figure diff to re-render treat them as new object, hence re-render) - does a similar concept not apply to compose? I see from your PR description that you evaluated creating a new object for control token when it's updated, did that not work/ you choose this method for alternate reasons..?``

Reply:
It worked but we were a bit worried about memory issues on creating new objects everytime a new ControlToken changes. Do you think that is a standard method across declarative UI and we should stick to it?

Reply:
I would incline towards following it if that works, it would make the codebase more readable and easy to reason with for anyone new. 
For the memory issue - the old instance should be dereferenced and eventually cleaned up from runtime instead of staying forever in app runtime, but you would defer on you to how compose fwd does it and what's the recommendation.

Reply:
Creating new object works out of the box. However, it is also desirable to manage theme via single object which could be maintained at app level. Hence, these changes required.

Reply:
I did not follow the explanation on how creating new objects prevents managing of theme via one object. Can you elaborate?

Reply:
Had a offline conversation, the real issue being designed is not related to memory. It is the current API support for control tokens update, and expected partner usages - where this model potentially simplifies the exposed API across multiple invocations.